BFF/v3/Vue/vue.client/vite.config.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs'
+import { fileURLToPath, URL } from 'node:url'
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import vueDevTools from 'vite-plugin-vue-devtools'
+
+const baseFolder =
+  process.env.APPDATA !== undefined && process.env.APPDATA !== ''
+    ? `${process.env.APPDATA}/ASP.NET/https`
+    : `${process.env.HOME}/.aspnet/https`
```

Comment:
This directory includes `dev-certs` on my machine. You could also just load the `.pfx` found in the directory and this would work.

![CleanShot 2025-03-31 at 08 12 50@2x](https://github.com/user-attachments/assets/05831afd-2fa0-45b8-900c-7c5031d10d7f)